docs/Installation-Windows.md

```diff
@@ -0,0 +1,190 @@
+# Installing ML-Agents for Windows
+
+ML-Agents supports Windows 10. While it might be possible to run ML-Agents using other versions of Windows, we only test Windows 10. (We also do not test running Windows on a VM like Bootcamp or Parallels).
+
+To use ML-Agents, you install Python and the required Python packages as outlined below. This guide also covers how set up GPU-based training (for advanced users). GPU-based training is not required for the v0.3 release of ML-Agents. However, training on a GPU might be required by future versions and features.
+
+## Step 1: Install Python via Anaconda
+
+<a href="https://www.anaconda.com/download/#windows" target="_blank">Download</a> and install Anaconda for Windows.  By using Anaconda, you can manage separate environments for different distributions of Python.  We **strongly** recommend using Python 3 as we do not guarantee supporting Python 2 in future releases.  In this guide, we are using Python version 3.6 and Anaconda version 5.1 ([64-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86_64.exe) or [32-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86.exe) direct links).
```

Comment:
We should change this to say we require Python 3.

Reply:
done